proto/api/v2/ai_integration_service.proto

```
@@ -0,0 +1,66 @@
+      put: "/v2/ai-integrations/{id}"
+      body: "*"
+    };
+  }
+
+  // DeleteAiIntegration deletes an AI integration by ID.
+  rpc DeleteAiIntegration(ResourceByID) returns (Empty) {
+    option (google.api.http) = {delete: "/v2/ai-integrations/{id}"};
+  }
+
+  // TestAiIntegration tests connectivity to an AI integration endpoint.
+  rpc TestAiIntegration(AiIntegration) returns (Empty) {
```

Buf (1.72.0)

[error] cannot find `Empty` in this scope

(COMPILE)